modules/plugin/jdbc/jdbc-oracle/src/test/java/org/geotools/data/oracle/sdo/SDOOnlineTest.java

```diff
@@ -68,54 +75,65 @@ protected void disconnect() throws Exception {
         super.disconnect();
     }
 
+    @Test
     public final void testGType() throws SQLException {
-        assertEquals(2003, SDO.gType(fixture.rectangle));
+        Assert.assertEquals(2003, SDO.gType(fixture.rectangle));
```

Comment:
Why not sure a static import for this one as well?

Reply:
The static import would be shadowed by the `assertEquals()` member method (https://stackoverflow.com/questions/6595980/import-static-does-not-work-when-the-class-has-methods-with-the-same-name-as-the).